docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -25,11 +25,28 @@ At the top right-hand side of the central pane, click on *Create Kubernetes cl
 A new pane named *Create {{k8s_management_service}} Shoot Cluster* slides over from the right-hand side of the browser.
 Type in a name for the new shoot cluster, and select a region.
 Also, select the version of Kubernetes the new cluster will be running.
-In the example below, we have chosen version 1.28.6, which, at the time of writing, was the latest supported in {{brand}}.
+In the example below, we have chosen version 1.28.7, which, at the time of writing, was the latest supported in {{brand}}.
 
 !["Create {{k8s_management_service}} Shoot Cluster" panel, showing options to type in a cluster name, select region and Kubernetes version](assets/create-shoot-1.png)
 
-Scroll down a bit until you bring the *Worker Groups* section into full view.
+Next, you may accept the proposed CIDR for the worker nodes or type in the one you prefer.
+
+![Accept the proposed CIDR for the worker nodes or type in the CIDR you prefer](assets/create-shoot-1.1.png)
+
+Alternatively, click on the drop-down menu named *Network for worker nodes* and select a pre-existing menu, one with a subnet and a router.
```

Comment:
I believe you meant to include the word "item" after "menu" here (or maybe say "item" _instead of_ "menu").

Reply:
Ugh, no. What I mean is that there's a drop-down menu named "Network for worker nodes" (that's the name of the drop-down menu, I kid you not). Also, I should replace the "pre-existing menu" with "pre-existing network". I'll modify the whole sentence.